openfisca_country_template/situation_examples/run.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
```

Comment:
According to [the doc](https://docs.python.org/3/howto/unicode.html), “Python supports writing source code in UTF-8 by default”, are you sure this is necessary?